lib/diamonds.ts

```diff
@@ -0,0 +1,327 @@
+import {
+  IDiamondReadable,
+  IDiamondWritable,
+} from '@solidstate/typechain-types';
+import { Contract, constants } from 'ethers';
+
+const Table = require('cli-table3');
+
+type Diamond = IDiamondReadable & IDiamondWritable;
```

Comment:
Avoid the composite type and use only the minimum type where necessary.  For example, `addUnregisteredSelectors` doesn't need `IDiamondWritable`.

Reply:
https://github.com/solidstate-network/solidstate-solidity/pull/158/commits/047964efc5ae891831a15dd38d09cd2ca767c831